docs/notebooks/ExB_drift.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
<p>This sort of duplicates the functionality of https://docs.plasmapy.org/en/latest/notebooks/particle_stepper.html, but as I wrote in the docs - highly unstable code etc etc, and at this point in time I don't think it's comfortable enough to use here rather than this short didactic example. So - I do like this; we could maybe add a note that this is a simplified implementation of https://docs.plasmapy.org/en/latest/notebooks/particle_stepper.html for now; and once I've refactored that one, I'll gyrate back here and see whether it makes sense to replace it :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/971/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Thanks for pointing this out. My understanding is that the particle tracker class uses the Boris algorithm to evolve a particle's position in prescribed E and B fields. Therefore, one could solve the problem using that class (as is done in the particle stepper notebook) but it would not require solving for the drift velocity analytically. Both implementations should yield the same result for this problem, but the Boris algorithm is much more widely applicable because it doesn't require an analytical solution for the velocity. I added a note explaining this at the end of the notebook and pointing users to the particle stepper notebook for comparison.  </p>